packages/ui/typedoc.json

```diff
@@ -1,5 +0,0 @@
-{
-  "entryPoints": ["src/Common.ts"],
```

Comment:
I think we still need this for generating the typedoc on the server?

Reply:
my bad! wrong command to remove `ui-type-doc...`
It was in textField when i merged it. 
Revert `typedoc` removed `ui-type-doc...`